target_chains/ethereum/contracts/forge-test/utils/PythTestUtils.t.sol

```diff
@@ -74,9 +78,87 @@ abstract contract PythTestUtils is Test, WormholeTestUtils {
         uint64 prevConf;
     }
 
+    struct PriceFeedMessage {
+        bytes32 priceId;
+        int64 price;
+        uint64 conf;
+        int32 expo;
+        uint64 publishTime;
```

Comment:
this is actually an int64.

Reply:
This particular thing is fine (because we use two's complement). I think we had this inconsistency from the past.. the reason is that `block.timestamp` on evm is unsigned.

I might fix the message format here though, and still keep the final PriceFeed exposed to the users as it is (`uint(256)`)

Reply:
oh i see. i think it's fine to leave it as uint64, maybe worth a comment though.